src/App.tsx

```diff
@@ -1,61 +1,114 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import { todo } from './components/types';
 
 export const App = () => {
+  const editedTodos = todosFromServer.map(todo => ({
+    ...todo,
+    user: usersFromServer.find(user => user.id === todo.userId) || {
+      id: 0,
+      name: 'Unknown User',
+      username: 'unknown',
+      email: 'unknown@example.com',
+    },
+  }));
+  
+  const [todos, setTodos] = useState<todo[]>(editedTodos)
```

Comment:
The `useState` hook is used correctly to initialize the `todos` state with `editedTodos`. However, ensure that the `todo` type imported from `./components/types` includes the `user` property as optional, which is correctly done in the `types.ts` file.